lib/browser/browser.ts

```diff
@@ -183,6 +199,11 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
 
     this.emitter = new Emitter();
     this.tokenManager = new TokenManager(this, args.tokenManager);
+    this.authStateManager = new AuthStateManager(this);
+    
+    // authService instance for downstream SDKs existing public APIs
+    // TODO: deprecate
+    this.authService = new AuthService(this);
```

Comment:
It makes sense while refactoring, but if possible, it would be best to absorb all these methods into the main "OktaAuth" instance so there is no separate "authService"

Reply:
I was planning to keep the `authService` to make the migration process from the downstream SDKs easier. like in (https://github.com/okta/okta-oidc-js/pull/869/files#diff-263be45c1cad52fdcc98556999612ea8R49), react-sdk can just and `oktaAuth` in context and still keeping expose `authService` without any change. 
Then we can plan on deprecating the `authService` filed (in auth-js) in next release, maybe it would make sense to add warning message in method/contructor of AuthService.

Reply:
In my opinion the migration is easier if there is a single "service" object that contains all expected methods. I cannot explain the relationship between "oktaAuth" and "authService", except to say "oktaAuth" is a sort of namespace. If this is the direction we WANT to go, then we would not deprecate "authService" we would move all "service" or API methods onto it and leave "oktaAuth" as only a namespace that contains other objects.  This seems a little heavy to me. If we added the methods on authService directly to OktaAuth, and then treated OktaAuth as the authService, what would break?

Reply:
right, the current relationship is only like `oktaAuth` composes a service util for the downstream to use (weak). I actually wasn't planning this field (oktaAuth.authService) to be used by any clients other than okta owned downstream SDKs, that's mostly the reason why I don't want to keep all the methods in the global space. How about make it as private field (_authService), then only use it in the OKTA downstream SDKs. Later on, we can move whatever needed under oktaAuth (I won't expect too many methods to be moved).

Reply:
What would be the problem with other clients using these methods? I think normalizing the API goes both directions. Downstream SDKs get some methods they haven't seen before (getWithPopup, tx, etc.). And users of auth-js get some stuff they haven't seen before (isAuthenticated, etc.). Most methods are just **aliases** or convenience methods (getUser, getTokenManager, loginRedirect), which we can individually deprecate or rename in the future.

Reply:
I still feel `OktaAuth` (auth-js instance) and `AuthService` are different objects. Mix them together may cause confusion in the downstream SDKs' docs, which we need to explain what oktaAuth is, then how to use `AuthService` methods from `oktaAuth`, then link docs together. 

But with a private field, AuthService is still AuthService, no concept switch, and basically no readme change from the downstream SDKs. What we need to do is gradually deprecate it.

Reply:
We will definitely have to explain what `oktaAuth` is and provide documentation (link to okta-auth-js). What I have a harder time explaining is what is `AuthService` (distinct from this `oktaAuth` instance). "Service" makes the most sense in Angular where "service" is a concept and the service's singleton instance can be dependency injected. But React doesn't have a unique concept of "service". We are providing an object instance which contains all API methods, which we expose as the variable `auth` via the context. Ideally, this `auth` object **is** the `oktaAuth` instance and **all** methods are available **directly** on it. In Angular, ideally the service that we inject **is** the `okta-auth-js` instance with no additional wrapping or indirection. We could refer to this instance of `okta-auth-js` as the "auth service", I think this terminology makes sense because it is following the pattern of being a widely-available singleton object instance which exposes API methods. Since it has all the same methods user's expect, there should be very little confusion. The "auth service" is an instance of `OktaAuth` from `@okta/okta-auth-js` library. In angular it can be dependency injected, in React it is passed by context. In either case it can created explicitly by the user, or implicitly from config.

Reply:
I was planning to expose both `oktaAuth` and `authService` at the same time, then it can make the doc work in downstream SDKs easier. But looks like it's your concern per how to distinguish them. That makes sense to me. 
I'll put methods in the `oktaAuth` scope.